Python/dynamic/SS.py

```diff
@@ -274,78 +280,60 @@ def function_to_minimize(chi_params_scalars, chi_params_init, params, iterative_
             dictionary[key] = locals()[key]
         pickle.dump(dictionary, open("OUTPUT/Saved_moments/SS_init_solutions.pkl", "w"))
     if (chi_params_init <= 0.0).any():
+        # In case the minimizer doesn't respect the bounds given
         output += 1e14
+    # Use generalized method of moments to fit the chi's
     weighting_mat = np.eye(2*J + S)
     scaling_val = 100.0
     value = np.dot(scaling_val * np.dot(output.reshape(1, 2*J+S), weighting_mat), scaling_val * output.reshape(2*J+S, 1))
     print 'Value of criterion function: ', value.sum()
     return value.sum()
 
-def run_steady_state(ss_parameters, iterative_params, get_baseline=False):
+def run_steady_state(ss_parameters, iterative_params, get_baseline=False, calibrate_model=False):
     '''
     ------------------------------------------------------------------------
-        Run SS in various ways, depending on the stage of the simulation
+        Run SS
     ------------------------------------------------------------------------
     '''
 
     if get_baseline:
         # Generate initial guesses for chi^b_j and chi^n_s
         chi_params = np.zeros(S+J)
-        if DATASET == 'REAL':
-            chi_params[0:J] = np.array([2, 10, 90, 350, 1700, 22000, 120000])
-            chi_n_guess = np.array([47.12000874 , 22.22762421 , 14.34842241 , 10.67954008 ,  8.41097278
-                                    ,  7.15059004 ,  6.46771332 ,  5.85495452 ,  5.46242013 ,  5.00364263
```

Comment:
Looks like you were able to get rid of this initial guess for chi_n altogether - is that correct?